samples/Verzeichnis C/Beispieldaten.txt

```diff
@@ -0,0 +1,77 @@
+configure network
```

Comment:
Bitte für diese Files einen Phaserer erstellen der das File ca. so in csharp Klassen umsetzt damit die Verarbeitung einfacher wird. Dafür wäre vermutlich ein seperater Issue besser um diesen hier nicht unnötig zu blockieren.

```csharp
class Network{
   public AccessList AccessList {get;set;}
   public NetworkDevice NetworkDev {get;set;}
   public NetworkInterface NetworkInterface {get;set;}
}
```


```csharp
public class AccessList {
   public bool UseSpecificInterface {get;set;}
   public string NetworkInterfaceName {get;set;}
   public AllowType AllowType {get;set;}
   public int PacketSize {get;set;}
}
```


```csharp
enum AllowType{
   allow,
   denie,
   maybe
}
```